app/views/course/levels/index.html.slim

```diff
@@ -1,8 +1,14 @@
 h1 = t('.header')
+
+- if can?(:create, Course::Level)
+  = new_button([current_course, :level]) { t('.create') }
```

Comment:
It shows an icon otherwise. Text would be clearer. Where should I put the text then?